lib/galaxy/model/__init__.py

```diff
@@ -3088,6 +3090,7 @@ def _source_dataset_chain(dataset, lst):
 
     @property
     def creating_job(self):
+        # TODO this should work with `return self.dataset.job` (revise failing unit tests)
```

Comment:
Only once we have a migration script that adds this. Before that you'd want to do `if self.dataset.job: return self.dataset.job` or something like that, then run through the rest.

Reply:
Thanks! I'll try this.